persistence-modules/hibernate-enterprise/src/main/java/com/baeldung/hibernate/pojo/PolygonEntity.java

```diff
@@ -1,10 +1,10 @@
 package com.baeldung.hibernate.pojo;
 
-import com.vividsolutions.jts.geom.Polygon;
+import org.locationtech.jts.geom.Polygon;
```

Comment:
I am not sure what we are trying to do here, but we have a Polygon class in the java itself https://docs.oracle.com/javase/7/docs/api/java/awt/Polygon.html

Reply:
I will try, but I wanted to keep the same dependency. The reason here is that because the package was changed
https://stackoverflow.com/questions/70660454/migrating-com-vividsolutions-jts-to-org-locationtech-jts-still-complaining-about

Reply:
Polygon can be used from java but some of the methods from hibernate spatial will not be compatible with that Java Object (Polygon)

Reply:
Let's not change this then